src/script.js

```diff
@@ -16,97 +16,92 @@ window.onload = () => {
 
 	const canvas = element;
 	const ctx = canvas.getContext('2d');
-	let x = (canvas.width / 2) - 50;
-	let y = 0;
-	let pointing = "down";
-	const theArrow = canvas.getContext('2d');
+
+	let movingArrow = {
```

Comment:
Just nitpicking but should be const since not changing the objects value

Reply:
You do have a point! I don't know why I let it as let and not const. Thanks! It's changed